tests/wpunit/Tribe/Body_ClassesTest.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace Tribe;
+
+use Tribe__Cache as Cache;
+
+class ClassesTest extends \Codeception\TestCase\WPTestCase {
+	/**
+	 * @return Classes
+	 */
+	protected function make_instance() {
+		return Classes::instance();
+	}
+
+	public function create_classes() {
+		$classes = $this->make_instance();
```

Comment:
The `make_instance` pattern is one I use when the class construction is not trivial, it is here.
Replacing one line with one line does slim code and increases the abstraction level (now I have to look up the `make_instance` method to know what I'm getting).
Just replace any use of `make_instance` with:
```suggestion
		$classes = new Body_Classes;
```

And, I think it's `Body_Classes`, not `Classes`, right?